src/App.jsx

```diff
@@ -1,9 +1,63 @@
-import React from 'react';
+import React, { useState } from 'react';
+
+import List from './List';
+import Input from './Input';
 
 export default function App() {
+  const [state, setState] = useState({
+    id: 0,
+    name: '',
+    category: '',
+    address: '',
+    restaurants: [],
+  });
+
+  const {
+    name,
+    category,
+    address,
+    restaurants,
+  } = state;
+
+  function handleChange(event) {
+    const { name: targetName, value } = event.target;
```

Comment:
`targetName`으로 한번 더 맵핑하신 이유가 있나요?

```suggestion
    const { name, value } = event.target;
```
이렇게만 해줘도 target의 name 정보인걸 알 수 있을 것 같아요.

Reply:
넵 이유가 있습니다.

윗 줄에서 상태를 구조분해할 때

```jsx
const { name, category, address, restaurants} = state;
```
이미 name이라는 키워드를 사용해서 eslint에서 잡아냈기 때문입니다.

![image](https://user-images.githubusercontent.com/8105528/108504477-467dba00-72f9-11eb-9654-42579131b40f.png)